frontend/src/store/modules/environment.ts

```diff
@@ -92,16 +93,10 @@ const actions = {
     commit("upsertEnvironmentList", environmentList);
 
     for (const environment of environmentList) {
-      await dispatch(
-        "policy/fetchPolicyByEnvironmentAndType",
-        {
-          environmentId: environment.id,
-          type: "bb.policy.pipeline-approval",
-        },
-        {
-          root: true,
-        }
-      );
+      await usePolicyStore().fetchPolicyByEnvironmentAndType({
```

Comment:
I'm not sure about whether we can use [Using a store outside of a component](https://pinia.vuejs.org/core-concepts/outside-component-usage.html#using-a-store-outside-of-a-component)

need more tests.

Reply:
Yes, we can.
We call `app.use(pinia)` very early (before any `useXXStore`) and we call `useXXStore` in component setup scope.
We must NOT call `useXXStore` in any module scope.
I've tested this module via UI and it works.

Reply:
TIL